Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations.yml

```
@@ -0,0 +1,63 @@
+commonfields:
+  id: QRadarGetOffenseCorrelations
+  version: -1
+name: QRadarGetOffenseCorrelations
+script: ''
+type: python
+tags:
+- QRadar
+comment: Return the QRadar offense correlations if exist in logs.
+system: true
+args:
+- name: offenseID
+  required: true
+  default: true
+  description: The offense ID to query.
+  defaultValue: ${incident.labels.id}
+- name: startTime
+  required: true
+  description: query from startTime (in timestamp format).
+  defaultValue: ${incident.labels.start_time}
+- name: include_null_correlations
+  required: true
+  description: Include Null CRE Names
+  defaultValue: "false"
+- name: headers
+  description: Table headers
+- name: range
+  description: Number of results in return
+- name: interval
+  description: Interval to poll for results. Default is 10 seconds.
+- name: timeout
+  description: Timeout in seconds. Default is 10 minutes.
+- name: fields
+  description: Property fields to include in results
+outputs:
+- contextPath: QRadar.Correlation
+  description: The QRadar offense correlations
+- contextPath: QRadar.Correlation.QID
+  description: The correlation QID identifier
+- contextPath: QRadar.Correlation.CREName
+  description: The correlation name
+- contextPath: QRadar.Correlation.CREDescription
+  description: The correlation description
+- contextPath: QRadar.Correlation.SourceIP
+  description: The correlation source IP
+- contextPath: QRadar.Correlation.DestinationIP
+  description: The correlation destination IP
+- contextPath: QRadar.Correlation.Category
+  description: The correlation high level category
+- contextPath: QRadar.Correlation.Username
+  description: The correlation username
+- contextPath: QRadar.Correlation.StartTime
+  description: The correlation start time
+- contextPath: QRadar.Correlation.CategoryID
+  description: 'The correlation category id '
+- contextPath: QRadar
+  description: QRadar context output
+scripttarget: 0
+dependson:
+  must:
+  - qradar-searches
+tests:
+- No test
```

Your build currently fails due to:
```
You've failed to provide tests for:
QRadar - Get offense correlations
```
I believe committing this change should fix it
```suggestion
  - No test
```